examples/jetstream-replay/config.toml

```
@@ -0,0 +1,11 @@
+[source]
+archive_url = "https://api.old-faithful.net"
+epoch = 800
+threads = 4
+reorder_buffer_size = 1000
+slot_timeout_secs = 30
```

Can you please adjust serde attributes for dash case?

```suggestion
archive-url = "https://api.old-faithful.net"
epoch = 800
threads = 4
reorder-buffer-size = 1000
slot-timeout-secs = 30
```